addons/iot_box_image/overwrite_before_init/etc/init_image.sh

```diff
@@ -84,6 +85,33 @@ pip() {
   fi
   pip3 \"\$1\" \"\$2\" --break-system-package \$additional_arg
 }
+
+devtools() {
+  if [[ -z \"\$1\" || -z \"\$2\" || \"\$1\" == \"-h\" ]]; then
+    echo 'Usage: devtools <enable/disable> <general/actions> [action name]'
+    echo ''
+    echo 'Only provide an action name if you want do enable/disable a specific device action.'
+    echo 'If no action name is provided, all actions will be enabled/disabled.'
+    return 1
+  fi
+  write_mode
+  if ! grep -q '^\[devtools\]' /home/pi/odoo.conf; then
+    printf '\n[devtools]\n' >> /home/pi/odoo.conf
+  fi
+  if [ \"\$1\" == \"disable\" ]; then
+    if [[ -z \"\$3\" ]]; then
+      value=\"*\"
+    else
+      value=\"\$3\"
+    fi
+    devtools enable \$2
+    write_mode
+    sed -i \"/^\[devtools\]/a\\\\\$2 = \$value\" /home/pi/odoo.conf
+  elif [ \"\$1\" == \"enable\" ]; then
+    sed -i \"/\[devtools\]/,/\[/{/\$2 =/d}\" /home/pi/odoo.conf
+  fi
+  read_mode
+}
```

Comment:
I'd say that for this to be less error-prone we can also limit the accepted arguments to:
1) `enable/disable`
2) `general/actions`

Otherwise we can end up with things like 

```
[devtools]
blabla = *
```